test/e2e/fixture-builder.js

```diff
@@ -115,9 +115,6 @@ function onboardingFixture() {
           [ETHERSCAN_SUPPORTED_CHAIN_IDS.MOONRIVER]: true,
           [ETHERSCAN_SUPPORTED_CHAIN_IDS.GNOSIS]: true,
         },
-        showTestNetworks: false,
-        smartTransactionsOptInStatus: true,
-        tokenNetworkFilter: {},
```

Comment:
if you scroll up some lines above, you'll see that these properties are already defined inside the `preferences` object (correct) -- these ones are the ones causing errors and wallet freeze.


See entry https://github.com/MetaMask/metamask-extension/pull/31521/files#diff-9e631c0f28d0aae4b92ffe878682c5e8479851bce192a7a4e8c6987014c96a61L84